seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/SourceSeaTunnelTask.java

```diff
@@ -76,10 +79,21 @@ public void init() throws Exception {
                             + startFlowLifeCycle.getClass().getName());
         } else {
             SeaTunnelDataType sourceProducedType;
+            List<TablePath> tablePaths = new ArrayList<>();
             try {
                 List<CatalogTable> producedCatalogTables =
                         sourceFlow.getAction().getSource().getProducedCatalogTables();
                 sourceProducedType = CatalogTableUtil.convertToDataType(producedCatalogTables);
+                tablePaths =
+                        producedCatalogTables.stream()
+                                .map(CatalogTable::getTableId)
+                                .map(
+                                        tableIdentifier ->
+                                                TablePath.of(
+                                                        tableIdentifier.getDatabaseName(),
+                                                        tableIdentifier.getSchemaName(),
+                                                        tableIdentifier.getTableName()))
```

Comment:
```suggestion
                                        tableIdentifier ->
                                                tableIdentifier.toTablePath()
```

Reply:
.map(
            tableIdentifier ->
                    TablePath.of(
                            tableIdentifier.getDatabaseName(),
                            tableIdentifier.getSchemaName(),
                            tableIdentifier.getTableName())) 
change : 
.map(TableIdentifier::toTablePath)